google/resource_container_cluster.go

```diff
@@ -1063,6 +1090,32 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 		d.SetPartial("logging_service")
 	}
 
+	// Disable update support for now: https://issuetracker.google.com/74063492
```

Comment:
Is this the best way?  I feel like maybe we might want to just use regular git / github for this, pull it out into its own branch, and leave the PR open with a note that it should be merged when the bug is closed.  Commented-out-code makes me nervous.

Reply:
Yeah, that's a better idea. I'll do that now.